dashboard/test/ui/runner.rb

```diff
@@ -708,6 +713,8 @@ def cucumber_arguments_for_browser(browser, options)
   arguments += skip_tag('@webpurify') unless CDO.webpurify_key
   arguments += skip_tag('@pegasus_db_access') unless options.pegasus_db_access
   arguments += skip_tag('@dashboard_db_access') unless options.dashboard_db_access
+  arguments += skip_tag('@properties_encryption_key') if CDO.properties_encryption_key.blank?
+  arguments += skip_tag('@cloudfront_key') if CDO.cloudfront_key_pair_id.blank?
```

Comment:
For parallelism, would it make sense to do something like,

```suggestion
  arguments += skip_tag('@webpurify') if CDO.webpurify_key.blank?
  arguments += skip_tag('@pegasus_db_access') if options.pegasus_db_access.blank?
  arguments += skip_tag('@dashboard_db_access') if options.dashboard_db_access.blank?
  arguments += skip_tag('@properties_encryption_key') if CDO.properties_encryption_key.blank?
  arguments += skip_tag('@cloudfront_key') if CDO.cloudfront_key_pair_id.blank?
```

Or even,

```suggestion
  arguments += skip_tag('@webpurify') unless CDO.webpurify_key.present?
  arguments += skip_tag('@pegasus_db_access') unless options.pegasus_db_access.present?
  arguments += skip_tag('@dashboard_db_access') unless options.dashboard_db_access.present?
  arguments += skip_tag('@properties_encryption_key') unless CDO.properties_encryption_key.present?
  arguments += skip_tag('@cloudfront_key') unless CDO.cloudfront_key_pair_id.present?
```

?